### PR TITLE
Add changelog for containerd v1.4.4-1

### PR DESCRIPTION
**Pull request by fuweid**
Signed-off-by: Wei Fu <fuweid89@gmail.com>

### CODE REVIEW

**Comment by fuweid** [comment]
ping @thaJeztah @tonistiigi @tiborvass ptal thanks~

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+containerd.io (1.4.4-1) release; urgency=medium
```
Comment (thaJeztah) [inline debian/changelog:1]: Perhaps good to change this to `urgency=high` (I know we did that on previous security releases of upstream containerd)

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks! Had this one on my list for today, so this saves me some work 😅; left one comment

File: debian/changelog (hunk 1 above)
Comment (fuweid) [reply debian/changelog]: Make senses to me. Updated~ Thanks

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
Hmm.. I notice that https://github.com/docker/containerd-packaging/pull/221 was also merged, but looks like packages were not (yet) pushed to download.docker.com; I'll check with @tiborvass if that was intentional (and if we'll skip that version, or push both);

```bash
$ apt-cache madison containerd.io | head -n4
containerd.io |    1.4.3-1 | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
containerd.io |    1.3.9-1 | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
containerd.io |    1.3.7-1 | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
containerd.io |   1.2.13-2 | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
```

I suspect that relates to https://github.com/docker/containerd-packaging/pull/222#pullrequestreview-606056893

**Comment by fuweid** [comment]
> https://github.com/docker/containerd-packaging/pull/221

just FYI, containerd doesn't mention runc-rc93 in vendor.conf 😂

**Comment by thaJeztah** [comment]
> just FYI, containerd doesn't mention runc-rc93 in vendor.conf 😂

Yes, so there's two things currently; current code in this repo was not yet adjusted for the switch to `go.mod` in upstream (had that one on my list; just pushed a fix, but looks like I need to modify somethings; https://github.com/docker/containerd-packaging/pull/224)

Reason we were already moving to runc-rc93 (slightly ahead of upstream containerd) is that there were too many issues related to seccomp when using images with recent ubuntu versions as base image. Runc rc93 has this patch https://github.com/opencontainers/runc/pull/2750, which helps for those situations; we know some cloud providers also started to install runc rc93 (and some distros started to patch their packages to update to that version as well);

**Review by tiborvass — APPROVED** [review]

**Pull request merged**